data/korath/korath ships.txt

```diff
@@ -309,89 +309,842 @@ ship "Korath Chaser"
+	turret -80.5 191.5 "Korath Warder"
+	turret 80.5 191.5 "Korath Banisher"
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. This many-lobed ship has been adapted to carry fighters into large-scale raids, or mining craft into asteroid belts. Its radial arrangements of hangars allows fighters a maximum variety of safe approach patterns in combat."
+
+
+ship "Kasichara A'awoj"
+	sprite "ship/world-ship f"
+	thumbnail "thumbnail/world-ship f"
+	attributes
+		category "Space Liner"
+		cost 42290000
+		shields 44000
+		hull 49600
+		"required crew" 543
+		"bunks" 1867
+		"mass" 2105
+		"drag" 24
+		"heat dissipation" .43
+		"solar collection" 12.7
+		"fuel capacity" 600
+		"cargo space" 1052
+		"outfit space" 1038
+		"weapon capacity" 331
+		"engine capacity" 251
+		weapon
+			"blast radius" 400
+			"shield damage" 8000
+			"hull damage" 4000
+			"hit force" 12000
+	outfits
+		"Korath Banisher" 3
+		"Korath Warder" 3
+		"Korath Grab-Strike" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Fuel Processor" 2
+		"Brig"
+		"Luxury Accommodations"
+		"Microbot Defense Station" 23
+		"Korath Repeater Rifle" 372
+		
+		"Afterburner (Comet Class)"
+		"Reverser (Lunar Class)"
+		"Thruster (Stellar Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+	
 	engine -37 343 .8
-	engine 0 343 .9
 	engine 37 343 .8
-	turret -76 -157 "Korath Grab-Strike"
-	turret 76 -157 "Korath Banisher"
-	turret -59 -135 "Korath Warder"
-	turret 59 -135 "Korath Grab-Strike"
-	turret -61 -6 "Korath Banisher"
-	turret 61 -6 "Korath Warder"
-	turret -66 210 "Korath Warder"
-	turret 66 210 "Korath Banisher"
+	"reverse engine" -16 -336 0.9
+	"reverse engine" 16 -336 0.9
+	"reverse engine" 0 -303 0.5
+		over
+	turret -78 -233.5 "Korath Banisher"
+	turret 78 -233.5 "Korath Warder"
+	turret -78 9.5 "Korath Grab-Strike"
+	turret 78 9.5 "Korath Grab-Strike"
+	turret -73.5 41 "Korath Banisher"
+	turret 73.5 41 "Korath Warder"
+	turret -78 231.5 "Korath Warder"
+	turret 78 231.5 "Korath Banisher"
 	explode "tiny explosion" 30
 	explode "small explosion" 45
 	explode "medium explosion" 50
 	explode "large explosion" 40
 	explode "huge explosion" 50
 	"final explode" "final explosion large"
-	description "Korath World-Ships are massive nomadic habitats that hold all that is left of their people."
+	description "When the Korath were exiled, it was on these gargantuan 'world-ships' they first fled and lived on. Since the introduction of purpose-built city ships, these ancient vessels have mostly been decommissioned, turned into stations, or reconfigured into smaller, specialized vessels. Yet a few still ply the starlanes, crewed by those too restless to live on stations yet not bold enough to go on raids."
```

Comment:
```suggestion
	description "Originally designed as a a luxury habitat for those wealthy enough to escape living on the polluted surface of Korath worlds, when the Korath were exiled, it was on these gargantuan 'world-ships' they first crammed themselves into and fled to the core.  Nostalgically, the class was then renamed after the devastated and now forbidden Korath homeworld."
	description "
	Since the introduction of purpose-built city ships, these ancient vessels have mostly been decommissioned, turned into stations, or reconfigured into smaller, specialized vessels. Yet a few still ply the starlanes, crewed by those too restless to live on stations yet not bold enough to go on raids."
```